examples/categories.py

```diff
@@ -12,13 +12,22 @@
 
 from rosette.api import API, DocumentParameters, RosetteException
 
-
 def run(key, alt_url='https://api.rosette.com/rest/v1/'):
     """ Run the example """
-    categories_url_data = "https://onlocationvacations.com/2015/03/05/the-new-ghostbusters-movie-begins-filming-in-boston-in-june/"
+    categories_url_data = "https://onlocationvacations.com/2018/02/06/downton-abbey-exhibition-extended-april-2-nyc/"
```

Comment:
This may get washed out by [this](https://git.basistech.net/raas/rosette-api-example-data/blob/master/src/main/resources/examples.properties#L4).